### PR TITLE
fix(doc): add kernel exporter doc and fix some misc doc problem

### PR DESCRIPTION
**Pull request by yeasoon**

### CODE REVIEW

**Comment by yeasoon** [comment]
@haolongzhangm @Li-Ming-xin please help to review

File: doc/how-to-use.md [hunk 1]
```diff
@@ -8,16 +8,17 @@
...
+├── mgb-runner      : aux app, run mdl model with mgb naive kernel, used to check megcc correctness
+├── hako-to-mgb     : aux app, unpack hako model to mgb model
+├── mgb-to-tinynn   : core app, megcc compiler, compile mdl model to kernel and binded model
+└── kernel_exporter : aux app, export kernel C code of given kernel type 
 
 ```
 
 ## Use Example In Android
-- Download the precompiled megcc from [github](https://github.com/MegEngine/MegCC/releases/download/v0.1.2/megcc_release_0.1.2.tar.gz)
+- Download the precompiled megcc from [github](https://github.com/MegEngine/MegCC/releases/download/v0.1.3/megcc_release_0.1.3.tar.gz)
```
Comment (haolongzhangm) [inline doc/how-to-use.md:21]: 感觉这里可以直接写  https://github.com/MegEngine/MegCC/releases

File: doc/how-to-use-chinese.md [hunk 2]
```diff
@@ -10,15 +10,16 @@
...
+├── megcc-opt       : 辅助工具，主要展示 MegCC 定义的 Pass 或者 MLIR 中预定义的 Pass 的具体细节，以及用于 Debug。
+├── mgb-runner      : 辅助工具，用于直接使用 MegEngine 运行模型，用于和 MegCC Runtime 的计算结果进行对比，验证正确性。
+├── hako-to-mgb     : 辅助工具，用于将使用 hako 打包之后的模型转换为 MegEngine 对应的模型。
+├── mgb-to-tinynn   : 主要的 MegCC 编译工具，将编译 MegEngine 模型，并输出运行这个模型需要的 Kernel，以及对应优化之后的模型。
+└── kernel_exporter : 辅助工具，用于指定 kernel C 代码的导出。
 ```
 
 # 使用 MegCC 完成模型部署
-首先需要从 [github](https://github.com/MegEngine/MegCC/releases/download/v0.1.2/megcc_release_0.1.2.tar.gz) 上下载需要的 MegCC 发版包，然后解压这个压缩包：`tar -xvf megcc_release_*.tar.gz`。使用 MegCC 完成模型部署主要步骤有三个步：
+首先需要从 [github](https://github.com/MegEngine/MegCC/releases/download/v0.1.3/megcc_release_0.1.3.tar.gz) 上下载需要的 MegCC 发版包，然后解压这个压缩包：`tar -xvf megcc_release_*.tar.gz`。使用 MegCC 完成模型部署主要步骤有三个步：
```
Comment (haolongzhangm) [inline doc/how-to-use-chinese.md:22]: 感觉这里直接写 https://github.com/MegEngine/MegCC/releases

File: doc/how-to-use.md (hunk 1 above)
Comment (yeasoon) [reply doc/how-to-use.md]: done

File: doc/how-to-use-chinese.md (hunk 2 above)
Comment (yeasoon) [reply doc/how-to-use-chinese.md:22]: done

**Pull request merged**